src/Workspaces/Core/Portable/Remote/RemoteHostClient.cs

```diff
@@ -98,18 +114,10 @@ protected Session(PinnedRemotableDataScope scope, CancellationToken cancellation
 
             public abstract Task InvokeAsync(string targetName, params object[] arguments);
 
-            /// <summary>
-            /// All caller must guard itself from it returning default(T) which can happen if OOP is killed
-            /// unintentionally such as user killed OOP process.
-            /// </summary>
```

Comment:
Leave these doc comments for API consumers outside of Roslyn?  (they still seem accurate/useful)